src/utils/resources/vm/utils/source.ts

```diff
@@ -10,6 +14,25 @@ import { TemplateBootSource } from '@kubevirt-utils/resources/template/hooks/use
 
 import { getBootDisk, getVolumes } from './selectors';
 
+export const getPVCSourceOrSourceRef = (
```

Comment:
Please add comments on this function similar to what we do for other functions in this file,
util functions under the `shared` directory should be well-defined and easy to understand